docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md

```diff
@@ -19,4 +19,22 @@ title: Pod Security Standards
 	* That updating a template will affect live clusters
 	* Who can edit. Including a template management role workaround with it's own warning/caveat.
 * How to upgrade
-	* Ensure that a replacement is in place before you upgrade and PSPs are off
\ No newline at end of file
+	* Ensure that a replacement is in place before you upgrade and PSPs are off
+
+
+### Running Rancher on a local cluster that is PSA restricted by default
```

Comment:
If this is the only header on the page, this should be the title. Or, if we add more information before the header, this section should be made into a second-level header. It seems like some content went missing and an early version of the draft got marked ready for review.